packages/grafana-data/src/field/fieldDisplay.ts

```diff
@@ -215,6 +215,7 @@ export const getFieldDisplayValues = (options: GetFieldDisplayValuesOptions): Fi
               ? () =>
                   fieldLinksSupplier({
                     calculatedValue: displayValue,
+                    valueRowIndex: 0,
```

Comment:
I'm not sure of the implications of hard coding the value row index here, @leeoniya do you have any insights on this?

Reply:
I've narrowed the change and added more commentary. The change is now more minimal.

On some level, choosing `valueRowIndex: 0` can be wrong. If you have more than one row in the data frame then the calc that was run may not be totally meaningful. Indeed this is referring to the entire data frame here when we talk about the `${__data}` macros and then scoped to the relevant row through this mechanism here.

The more minimal and correct thing I've now done to limit this change to apply only to data frames with a single row. Then there's no doubt that it's the right choice for the row index (it's the only reasonable choice :p).